website/server/libs/chatReporting/inboxChatReporter.js

```diff
@@ -30,8 +30,9 @@ export default class InboxChatReporter extends ChatReporter {
     const validationErrors = this.req.validationErrors();
     if (validationErrors) throw validationErrors;
 
+    // added .exec() to the end of line 35 as suggested by @paglias
```

Comment:
can we remove these comment? Comments should be used when they add extra information that might be useful to understand what the code is doing not to explain changes made as part of the PR